### PR TITLE
captcha v2 returns

### PR DESCRIPTION
**Pull request by yaronpx**

### CODE REVIEW

File: lib/px/utils/pxclient.lua [hunk 1]
```diff
@@ -315,6 +319,13 @@ function M.load(px_config)
             ngx_req_set_header('cookie', 'pxvid=' .. vid)
         end
 
+        local xff_header = ngx.req.get_headers()['X-Forwarded-For']
```
Comment (nitzanpx) [inline lib/px/utils/pxclient.lua]: why not to do this at `forward_to_perimeterx`

Comment (yaronpx) [reply lib/px/utils/pxclient.lua]: because it's only relevant for xhr calls

File: README.md [hunk 2]
```diff
@@ -57,6 +56,22 @@ a request is allowed to continue being processed. When the PerimeterX Enforcer d
...
 See the full [changelog](CHANGELOG.md) for all versions.
 
+To upgrade the PerimeterX Enforcer: 
+
+1. Copy the configuration file [(pxconfig.lua)](lib/px/pxconfig.lua) from the latest source version to your application.
+2. Modify the new configuration file, making sure that it reflects your current/old configuration.
+3. Copy the new file to `/usr/local/lib/lua/px/pxconfig.lua`. (default location). If you wish to change the directory, you must save the file to where the module was previously installed. 
+
+### Custom Block Pages
+
```
Comment (pxjohnny) [inline README.md]: Should be linked to https://console.perimeterx.com/docs/server_integration_new.html#custom-captcha-section as well imo...

Comment (yaronpx) [reply README.md]: it's not supported in this way yet...

**Pull request merged**